src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -505,6 +505,25 @@ class genUtils {
 
         return result;
     }
+    
+    static truncateDocumentId(documentId: string): string {
```

Comment:
one more small enhacement: instead of having chars to show as const, let's put this as parameter with default value.

Reply:
done